backend/tracim_backend/applications/share/config.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+from tracim_backend import CFG
+from tracim_backend.exceptions import ConfigurationError
+from tracim_backend.lib.utils.translation import translator_marker as _
+
+
+def load_config(app_config: CFG) -> CFG:
+    # share content email
+    app_config.EMAIL__NOTIFICATION__SHARE_CONTENT_TO_RECEIVER__TEMPLATE__HTML = app_config.get_raw_config(
+        "email.notification.share_content_to_receiver.template.html"
+    )
```

Comment:
A question about app_config: When do `app_config.get_raw_config(`: What happens if config not provided ?

If return None and it is a required config: not very stable.

Reply:
checking is done after loading. That's why there is both load_config and check_config method.
This allow to deal with some complex case where some config are depending on other and it make sense to be required only in some case.